cmd/kubelet/app/server_bootstrap_test.go

```diff
@@ -42,6 +42,7 @@ import (
 	certutil "k8s.io/client-go/util/cert"
 	capihelper "k8s.io/kubernetes/pkg/apis/certificates/v1"
 	"k8s.io/kubernetes/pkg/controller/certificates/authority"
+	"k8s.io/kubernetes/test/utils/ktesting"
```

Comment:
up to the rest of the package owners, but i think importing /test under any component packages like /cmd or /pkg might not be what we want.

moving this to k8s.io/utils would work as @pohly suggests.